nutils/_graph.py

```diff
@@ -68,7 +69,33 @@ def export_graphviz(self, *, fill_color: Optional[GraphvizColorCallback] = None,
                 for i, line in enumerate(src.split('\n'), 1):
                     print('{:04d}  {}'.format(i, line))
                 treelog.warning('graphviz failed for error code', status.returncode)
-            img.write(status.stdout)
+            graph = status.stdout
+            if image_type == 'svg':
+                graph = re.sub(b'<svg (.*?)>', insert_clickHandler, graph, count=1, flags=re.DOTALL)
+            img.write(graph)
+
+insert_clickHandler = rb'''<svg \1 onload="document.addEventListener('click', clickHandler)">
+<style>
+.highlight path { stroke: orange; stroke-width: 2; }
+.highlight polygon:first-of-type { fill: orange; }
+.highlight polygon { stroke: orange; }
+.highlight text { fill: white; }
+</style>
```

Comment:
Suggestion: With `g.edge, g.node { cursor: pointer }` you get a pointer if you hover above clickable elements.

Reply:
Ah, there is a semicolon missing after pointer. Sorry!

Reply:
fixed!